ikarus/controlroutines/pathfollowingtechnique.hh

```diff
@@ -16,54 +18,86 @@
 
 namespace Ikarus {
 
-  struct PathFollowingInformation {
+  struct ControlInformation {
```

Comment:
This should be in a seperate header an should be included by all controlroutines

Reply:
Done